packages/jest-runtime/package.json

```diff
@@ -25,7 +30,8 @@
     "testPathIgnorePatterns": [
       "/__tests__/[^/]*/.+"
     ],
-    "testEnvironment": "jest-environment-node"
+    "testEnvironment": "jest-environment-node",
+    "automock": false
```

Comment:
put that on top of the jest config please. alphabetize the config (I realize it wasn't alphabetized before).